doc/teehistorian.md

```diff
@@ -54,12 +54,45 @@ message ID.
 * DROP(-9): cid(int) reason(str) records that a player with cid left/was kicked/was dropped, on the engine level
 * CONSOLE_COMMAND(-10): cid(int) flags(int) cmd(str) num_args(int) args(str[num_args]) records that a console command cmd was executed by client id cid (not necessarily a player, might be a vote as well), with flags (distinguishes chat commands, etc.) with parameters args
 * EX(-11): uuid(uuid) size(int) data(raw[size]) records an extension message, identified by uuid and containing data
+  * introduced in [11.0.3](https://github.com/ddnet/ddnet/commit/6c378b972b70b0556d3b434b26baa0b9ffe490f1)
 
 The following extra messages are known right now:
 * TEST(teehistorian-test@ddnet.tw): is just a test message
+  * uuid: `6bb8ba88-0f0b-382e-8dae-dbf4052b8b7d`
+  * introduced in [11.0.3](https://github.com/ddnet/ddnet/commit/6c378b972b70b0556d3b434b26baa0b9ffe490f1)
```

Comment:
It would be nice if the links could describe where they point to, e.g.:

> Introduced in DDNet 13.2, [0d7872c79eaeb19b](https://github.com/ddnet/ddnet/commit/0d7872c79eaeb19b3fd08c39c013a1043db1fd9b).